### PR TITLE
Docs: Update ib validation microbenchmark metrics 

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Update ib validtion mirobenchmark metrics.

### CODE REVIEW

File: docs/user-tutorial/benchmarks/micro-benchmarks.md [hunk 1]
```diff
@@ -192,6 +192,22 @@ gpcnet-network-load-test: Select full system network tests run with four congest
 | {benchmark_name}/${test_title}_Two-sidedIncast(4096B)_${stat}       | bandwidth (MB/s/rank) | statistical values(Min, Max, Avg, 99%, 99.9%) obtained by all nodes use congestion 'Two-sided Incast (4096 B)' for congestion testing                                      |
 | {benchmark_name}/${test_title}_Alltoall(4096B)_${stat}              | bandwidth (MB/s/rank) | statistical values(Min, Max, Avg, 99%, 99.9%) obtained by all nodes use congestion 'Alltoall (4096 B)' for congestion testing                                              |
 | gpcnet-network-load-test/${test_title}_${network_test_algo}_${stat} | times(x)              | summary about congestion impact factor of every network test algorithm                                                                                                     |
+### `ib-traffic`
```
Comment (cp5555) [inline docs/user-tutorial/benchmarks/micro-benchmarks.md:195]: empty line before it.

**Review by cp5555 — APPROVED** [review]

File: docs/user-tutorial/benchmarks/micro-benchmarks.md [hunk 2]
```diff
@@ -192,6 +192,22 @@ gpcnet-network-load-test: Select full system network tests run with four congest
...
+Measure the InfiniBand performance under multi nodes' traffic pattern.
+
+The traffic pattern is defined in a config file, which is pre-defined for one-to-many, many-to-one and all-to-all patterns.
+Each row in the config is one round, and all pairs of nodes in a row run ib command simultaneously.
+
+#### Metrics
+                                                              
+| Metrics                    | Unit            | Description                                                                                                                                      |
+| -------------------------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| \<COMMAND>-\<LINE>-\<PAIR> | bandwidth(MB/s) | The average bandwidth of ib command(ib_write_bw, ib_send_bw, ib_read_bw) run between the \<pair>th node pair in the \<line>th line of the config |
```
Comment (abuccts) [inline docs/user-tutorial/benchmarks/micro-benchmarks.md:208]: could you use lower cases?

File: docs/user-tutorial/benchmarks/micro-benchmarks.md [hunk 3]
```diff
@@ -192,6 +192,22 @@ gpcnet-network-load-test: Select full system network tests run with four congest
...
+| \<COMMAND>-\<LINE>-\<PAIR> | time(usec)      | The max latency of ib command(ib_write_lat, ib_send_lat, ib_read_lat) run between the \<pair>th node pair in the \<line>th line of the config    |
```
Comment (abuccts) [inline docs/user-tutorial/benchmarks/micro-benchmarks.md:209]: a space before "("

**Review by abuccts — COMMENTED** [review]
there's build error, could you avoid using tags "<>"?

File: docs/user-tutorial/benchmarks/micro-benchmarks.md [hunk 4]
```diff
@@ -193,6 +193,23 @@ gpcnet-network-load-test: Select full system network tests run with four congest
...
+Measure the InfiniBand performance under multi nodes' traffic pattern.
+
+The traffic pattern is defined in a config file, which is pre-defined for one-to-many, many-to-one and all-to-all patterns.
+Each row in the config is one round, and all pairs of nodes in a row run ib command simultaneously.
+
+#### Metrics
+                                                              
+| Metrics                    | Unit            | Description                                                                                                                                      |
+| -------------------------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| ib-traffic/\${command}-\${line}-${pair} | bandwidth(MB/s) | The average bandwidth of ib command (ib_write_bw, ib_send_bw, ib_read_bw) run between the ${pair}th node pair in the ${line}th line of the config |
```
Comment (abuccts) [inline docs/user-tutorial/benchmarks/micro-benchmarks.md:209]: remove "\" before "$"

Comment (abuccts) [inline docs/user-tutorial/benchmarks/micro-benchmarks.md:209]: space before "(MB/s)"

File: docs/user-tutorial/benchmarks/micro-benchmarks.md [hunk 5]
```diff
@@ -193,6 +193,23 @@ gpcnet-network-load-test: Select full system network tests run with four congest
...
+| ib-traffic/\${command}-\${line}-${pair} | time(usec)      | The max latency of ib command (ib_write_lat, ib_send_lat, ib_read_lat) run between the ${pair}th node pair in the ${line}th line of the config    |
```
Comment (abuccts) [inline docs/user-tutorial/benchmarks/micro-benchmarks.md:210]: `time(usec)` -> `time (us)`
`${pair}th` -> `${pair}<sup>th</sup>`

**Review by abuccts — COMMENTED** [review]
"validtion" -> "validation" in title and description

**Review by abuccts — APPROVED** [review]
"mirobenchmark" -> "microbenchmark" in title and description

**Pull request merged**